manifests/params.pp

```diff
@@ -9,6 +9,9 @@
   $manage_package = true
   $manage_repo = true
   $package_ensure = present
+  $extpack_ensure = present
+  $extpack_version = $::virtualbox_version
+  $extpack_source = ''
```

Comment:
Let's leave the `extpack_source` parameter out. This way, it'll be set to `undef` instead of an empty string.